### PR TITLE
Return better error for issue #1502.

### PR DESCRIPTION
**Pull request by ericsage**
Signed-off-by: Eric Sage <eric.david.sage@gmail.com>

### CODE REVIEW

File: commands/ls.go [hunk 1]
```diff
@@ -87,6 +87,9 @@ func parseFilters(filters []string) (FilterOptions, error) {
 	options := FilterOptions{}
 	for _, f := range filters {
 		kv := strings.SplitN(f, "=", 2)
+		if len(kv) != 2 {
+			return options, fmt.Errorf("Unupported filter syntax, please use help ls for details on correct usage.")
```
Comment (nathanleclaire) [inline commands/ls.go]: You can probably just shorten this to "Unsupported filter syntax".  Consulting the help text is largely implied, I feel.

**Comment by nathanleclaire** [comment]
One small nit, otherwise looks good.

@hairyhenderson Does it LGTY?

**Comment by hairyhenderson** [comment]
Great! Thank you @ericsage. LGTM

**Comment by ericsage** [comment]
Thanks, I made the change to the usage. should be GTG.

**Comment by nathanleclaire** [comment]
Argh, wow, this is going to seem super pedantic, but I think that really should be a call to `errors.New`.  Sorry, I should have caught it before.  Can you update please?

**Comment by ericsage** [comment]
Done

**Comment by nathanleclaire** [comment]
Thanks!  LGTM
